HISTORY.rst

```diff
@@ -32,6 +32,9 @@ unreleased
 * Resource group for endpoint resources are no longer required - if not present, the resource group of the
   digital twins instance is used.
 
+**IoT Central updates**
```

Comment:
Can we place this under a 0.18.3 entry? This change just missed the 0.18.2 release.

Reply:
@digimaun i updated history